buildspec_clang.yml

```diff
@@ -14,6 +14,10 @@ phases:
       - echo $CODEBUILD_WEBHOOK_ACTOR_ACCOUNT_ID
       - echo $CODEBUILD_WEBHOOK_EVENT
       - echo $CODEBUILD_RESOLVED_SOURCE_VERSION
+      - pip install xarray
+      - echo $PATH
+      - echo $LD_LIBRARY_PATH
+      - echo $PYTHONPATH
```

Comment:
Do you want to leave these echos in or clean up?  I'm ok either way - leaving them in might be useful in future debugging.